src/hooks/useKnockoutExplorer.ts

```diff
@@ -364,11 +364,16 @@ function updateContextsFromPortalMessage(inputs: DataExplorerInputsFrame) {
     if (inputs.flights.indexOf(Flights.PKPartitionKeyTest) !== -1) {
       userContext.features.partitionKeyDefault2 = true;
     }
-    if (inputs.flights.indexOf(Flights.PhoenixNotebooks) !== -1) {
+    if (configContext.isPhoenixEnabled === true) {
       userContext.features.phoenixNotebooks = true;
-    }
-    if (inputs.flights.indexOf(Flights.PhoenixFeatures) !== -1) {
       userContext.features.phoenixFeatures = true;
+    } else {
+      if (inputs.flights.indexOf(Flights.PhoenixNotebooks) !== -1) {
+        userContext.features.phoenixNotebooks = true;
+      }
+      if (inputs.flights.indexOf(Flights.PhoenixFeatures) !== -1) {
+        userContext.features.phoenixFeatures = true;
+      }
```

Comment:
Are the old feature flags unused going forward? if so, remove them and all references to them

Reply:
We are still based of the flags